game_class.py

```diff
@@ -0,0 +1,63 @@
+class Game:
+    def __init__(self, board_size, who_starts = 'x'):
+        self.board_size = board_size
+        self.board = self._create_board()
+        self.whose_turn_is_it = who_starts
+    
+    def _create_board(self):
+        list1 = []
+        board = []
+        for i in range(self.board_size):
```

Comment:
can you change i to something more verbose?

Reply:
Updated